### PR TITLE
Updated OnlinePstnGateway with new parameters

### PR DESCRIPTION
**Pull request by JensTrier**
Added descriptions for the 2 parameters Description and GatewayLbrEnabledUserOverride
Add a sentence to description for FailoverResponseCodes
Changed order of parameter description
Changed applicable to be Microsoft Teams

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: skype/skype-ps/skype/New-CsOnlinePSTNGateway.md [hunk 1]
```diff
@@ -101,22 +116,42 @@ Allows some codecs to be excluded when media is being negotiated between Media P
...
 Accept wildcard characters: False
 ```
 
+### -FailoverResponseCodes
+If Direct Routing receives any 4xx or 6xx SIP error code in response to an outgoing Invite the call is considered completed by default. (Outgoing in this context is a call
+from a Teams client to the PSTN with traffic flow: Teams Client -> Direct Routing -> SBC -> Telephony network). Setting the SIP codes in this parameter forces Direct Routing
+on receiving the specified codes try another SBC (if another SBC exists in the voice routing policy of the user). Find more information in the "Reference" section of "Phone
+System Direct Routing" documentation.
+
+Setting this parameter overwrites the default values, so if you want to include the default values, please add them to string string.
```
Comment (get-itips) [inline skype/skype-ps/skype/New-CsOnlinePSTNGateway.md:133]: is "please add them to the string" maybe?

File: skype/skype-ps/skype/New-CsOnlinePSTNGateway.md [hunk 2]
```diff
@@ -101,22 +116,42 @@ Allows some codecs to be excluded when media is being negotiated between Media P
...
+
+```yaml
+Type: Int
```
Comment (get-itips) [inline skype/skype-ps/skype/New-CsOnlinePSTNGateway.md:136]: Please review the parameter type

**Comment by yogkumgit** [comment]
@JensTrier Please review. Thanks

File: skype/skype-ps/skype/New-CsOnlinePSTNGateway.md (hunk 1 above)
Comment (JensTrier) [reply skype/skype-ps/skype/New-CsOnlinePSTNGateway.md]: Fixed

File: skype/skype-ps/skype/New-CsOnlinePSTNGateway.md (hunk 2 above)
Comment (JensTrier) [reply skype/skype-ps/skype/New-CsOnlinePSTNGateway.md]: Fixed

**Comment by JensTrier** [comment]
Updated both new and set to address the comments.

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**